app/src/main/java/org/stepic/droid/code/highlight/themes/CodeTheme.kt

```diff
@@ -9,5 +11,7 @@ data class CodeTheme (
         @ColorInt val lineNumberBackground: Int,
         @ColorInt val lineNumberStroke: Int,
         @ColorInt val lineNumberText: Int,
-        @ColorInt val selectedLineBackground: Int
+        @ColorInt val selectedLineBackground: Int,
+        @ColorInt val bracketsHighlight: Int = selectedLineBackground,
```

Comment:
it will intersect with `selectedLineBackground` and not visible. This colors is different in Android Studio (http://prntscr.com/gz17hr). Let's choose another default color